config/200-aggregation-admin-clusterroles.yaml

```diff
@@ -0,0 +1,170 @@
+# Copyright 2022 TriggerMesh Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# All sources used at TriggerMesh
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: triggermesh-sources-admin
+  labels:
+    rbac.triggermesh.io/sources-admin: "true"
+rules:
+- apiGroups:
+  - sources.knative.dev
+  - sources.eventing.knative.dev
```

Comment:
To my knowledge this API group does not exist.

Reply:
It doesn't anymore, but did back in the 0.13.x days: https://github.com/knative/docs/blob/v0.13.x/docs/reference/eventing/eventing.md